config.json

```diff
@@ -10,7 +10,6 @@
   },
   "blurb": "Ruby is a dynamic, open source programming language with a focus on simplicity and productivity. It has an elegant syntax that is natural to read and easy to write.",
   "version": 3,
-  "gitter": "ruby",
```

Comment:
Are we using this somewhere?  We could change it to `"forum":   "c/general/building-exercism/125"` to match the location of the forum.

Reply:
> I've confirmed that we do not use the gitter key in the config.json and it can safely be removed.

Thanks @kytrinyx